### PR TITLE
Introduce source-location annotations

### PR DESCRIPTION
**Pull request by Irma12**

### CODE REVIEW

**Review by iain-b — DISMISSED** [review]
This should fix #266 right ?

File: plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const entityMock = {
     namespace: 'default',
     annotations: {
       'backstage.io/managed-by-location':
+      'url:https://github.com/mcalus3/sample-service/blob/master/backstage5.yaml',
```
Comment (iain-b) [inline plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts:21]: That's a real username, perhaps remove it ? (I know it was there before)

Comment (Irma12) [reply plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts:21]: That is a good point, maybe we should consider making changes in all of the plugins containing it

**Comment by Irma12** [comment]
@iain-b Hopefully yes :)

File: plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts (hunk 1 above)
Comment (iain-b) [reply plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts:21]: No harm but no rush or urgency on it though.

**Pull request merged**